src/components/UI/Form/StyledInputContainer.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const InputContainer = styled.div`
+	text-transform: uppercase;
+	color: var(--text-color-dark);
+	width: 383px;
```

Comment:
Is it planned that your form has a specific width and height? It looks like your input are cut on the right side but that's maybe part of the design?

Reply:
Hey Anita, this is part of the design. I showed it to the others, I will change it so there is no confusing